staff/carlos-calvo/precourse/pasapalabra/Pasapalabra.html

```diff
@@ -3,8 +3,8 @@
     <head>
```

Comment:
Yey!! this file shouldn't be committed in that branch! You should delete it from here and commit it in the precourse branch.